test/lambdakube/core_test.clj

```diff
@@ -1307,6 +1307,7 @@ spec:
 ")
 
 ;; `kube-apply` takes a string constructed by `to-yaml` and a `.yaml`
+;; optionally supply namespace and kube config parameters.
```

Comment:
This line seems to be out of place. It is in the middle of a sentence...

I would move it to just before line 1323, to describe the new test you created.

Reply:
okay made those changes.